internal/provider/kubernetes/kubernetes.go

```diff
@@ -148,11 +148,11 @@ func New(ctx context.Context, restCfg *rest.Config, svrCfg *ec.Server, resources
 			},
 		}
 	}
-
 	// ProxyTopologyInjector is the only component that interacts with Pods.
 	mgrOpts.Cache.ByObject[&corev1.Pod{}] = cache.ByObject{
 		Label: labels.SelectorFromSet(proxy.EnvoyAppLabel()),
 	}
+	mgrOpts.Cache.DefaultTransform = cache.TransformStripManagedFields()
```

Comment:
How does this interact with other systems applying resource changes like Kyverno or any tool that makes use of managed fields?

Reply:
this won't affect how kube-apiserver store the data.

Reply:
Chatted with @arkodg and this makes sense. It only affects status subresource update events.